utils/swanmon/test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+swanmon help
```

Comment:
This is really simple. Don't you want to rather check version as it is done in this repository for many packages? Based from your output in OP, the command version is there.

Reply:
The tool itself does not have a version command. `swanmon version` simply returns the underlying strongswan version, if started.